File: pset1/mario-more.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h; //declares the variable
    
    // prompting the user for a value between 1 and 8
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    
    for (int i = 0; i < h; i++) // number of lines
    {
        for (int s = h - 1; s > i; s--) // align right
        {
            printf (" ");
        }
        for (int w = 0; w <= i; w++) // prints first hashes
        {
            printf ("#");
        }
        
        printf ("  "); // prints the space between columns
        
        for (int w = 0; w <= i; w++) //prints second hashes
        {
            printf ("#");
        }
        
        printf ("\n"); // breaks line
    }
    
}

File: pset6/mario-more.py
from cs50 import get_int

height = get_int("Height: ")

while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    for j in range((height-1), i, -1):
        print(" ", end="")

    for w in range(i+1):
        print("#", end="")

    print("  ", end="")

    for z in range(i+1):
        print("#", end="")

    print()
